packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1441,4 +1441,25 @@ describe('Dropdown', () => {
       expect(stopPropagation).toBeCalledTimes(1)
     })
   })
+
+  describe('renderSelectedItem', () => {
+    it('calls renderSelectedItem in multiple selection', () => {
+      const renderSelectedItem = jest.fn((SelectedItem, props) => (
+        <SelectedItem {...props} header={`${props.header} (selected)`} />
+      ))
+      const wrapper = mountWithProvider(
+        <Dropdown multiple items={items} renderSelectedItem={renderSelectedItem} />,
+      )
+      const triggerButton = getTriggerButtonWrapper(wrapper)
+
+      triggerButton.simulate('click')
+      const firstItem = getItemAtIndexWrapper(wrapper)
+      firstItem.simulate('click', { nativeEvent: { stopImmediatePropagation: jest.fn() } })
```

Comment:
```suggestion
      firstItem.simulate('click', { nativeEvent: { stopImmediatePropagation: _.noop } })
```

_nit_, you can use `_.noop` from Lodash.